02-microarray/ortholog-mapping_microarray_01_ensembl.Rmd

```diff
@@ -0,0 +1,398 @@
+
+# Using a different refine.bio dataset with this analysis?
+
+If you'd like to adapt an example analysis to use a different dataset from [refine.bio](https://www.refine.bio/), we recommend placing the files in the `data/` directory you created and changing the filenames and paths in the notebook to match these files (we've put comments to signify where you would need to change the code).
+We suggest saving plots and results to `plots/` and `results/` directories, respectively, as these are automatically created by the notebook.
+From here you can customize this analysis example to fit your own scientific questions and preferences. 
+
+***
+
+<!-- Do not delete this line --> <a name="analysis" style="padding-top:56px;margin-top:-56px;">&nbsp;</a>
+
+
+# Ortholog Mapping - Microarray
+
+## Install libraries
+
+See our Getting Started page with [instructions for package installation](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#what-you-need-to-install) for a list of the other software you will need, as well as more tips and resources.
+
+Attach the package we need for this analysis.
+
+```{r}
+# We will need this so we can use the pipe: %>%
+library(magrittr)
+```
+
+## Import data from HGNC
+
+The HUGO Gene Nomenclature Committee (HGNC) assigns a unique and ideally meaningful name and symbol to every human gene.
+The [HGNC database](https://www.genenames.org/) currently contains over 39000 public records containing approved human gene nomenclature and associated gene information [@Gray2015].
+
+The [HGNC Comparison of Orthology Predictions (HCOP)](https://www.genenames.org/tools/hcop/) is a search tool that combines orthology predictions for a specified human gene, or set of human genes, by eggNOG, Ensembl Compara, HGNC, HomoloGene, Inparanoid, NCBI Gene Orthology, OMA, OrthoDB, OrthoMCL, Panther, PhylomeDB, PomBase, TreeFam and ZFIN, allowing users to obtain and compare orthology data [@Wright2005].
+The prediction with the most number of databases which concur is considered the most reliable.
+HCOP was originally designed to show orthology predictions between human and mouse, but has been expanded to include data from 18 genomes, including zebrafish, which we will use in this notebook [@hcop-help].
+
+In the next chunk, we have adapted code from the [`hcop` package R script](https://github.com/stephenturner/hcop/blob/master/data-raw/hcop.R) to retrieve the zebrafish and human mapped data from the HGNC database [@Turner2018].
+
+```{r}
+# Read in the data from HGNC
+zebrafish <-
+  paste0(
+    "ftp://ftp.ebi.ac.uk/pub/databases/genenames/hcop/",
+    "human_",
+    "zebrafish", # Replace with the organism you have data for
+    "_hcop_fifteen_column.txt.gz"
+  ) %>%
+  data.table::fread() %>%
+  set_names(names(.) %>% gsub("_gene$", "", .)) # Removing some extra wording in some of the column names
+```
+
```

Comment:
I would print out a preview of the first few lines of `zebrafish` so users can see what it looks like since this is a new thing. (aka always be telling them to look at the data to get familiar)